lib/ohai/plugins/network.rb

```diff
@@ -54,7 +54,7 @@ def sorted_ips(family = "inet")
     ipaddresses.sort_by do |v|
       [ ( scope_prio.index(v[:scope]) || 999999 ),
         128 - v[:ipaddress].prefix.to_i,
-        ( family == "inet" ? v[:ipaddress].to_u32 : v[:ipaddress].to_u128 ),
+        v[:ipaddress].respond_to?(:to_u32) ? v[:ipaddress].to_u32 : v[:ipaddress].to_u128,
```

Comment:
I can't see any reason to just just make this:
```suggestion
        v[:ipaddress].to_i
```
@lamont / @tas50  am I missing something?